app/managers.py

```diff
@@ -0,0 +1,51 @@
+import sqlite3
+
+from models import Actor

```

Comment:
The import statement for `Actor` should include the correct module path: `from app.models import Actor` to match the project structure.